packages/react-instantsearch/examples/autocomplete/package.json

```diff
@@ -8,8 +8,8 @@
     "react-autosuggest": "^7.0.1",
     "react": "^15.3.2",
     "react-dom": "^15.3.2",
-    "react-instantsearch": "file:../../dist",
-    "react-instantsearch-theme-algolia": "file:../../../react-instantsearch-theme-algolia"
+    "react-instantsearch": "latest",
```

Comment:
Never tried yet, I did not yet address the problem of "I want to develop on this example with auto reload of react-instantsearch code". It was not handled before this commit too.

We will need to link dependencies via a script maybe.